src/Map/Navigation/Path.java

```diff
@@ -268,7 +278,7 @@ private void createPathRandom(MapNode start, MapNode end, boolean vFlag) throws
      * @param vFlag
      * @throws PathFindingException
      */
-    private void createPathBreadthFirst(MapNode start, MapNode end, boolean vFlag) throws PathFindingException {
+    private void createPathBreadthFirst(MapNode start, MapNode end, boolean vFlag, boolean useStairs) throws PathFindingException {
```

Comment:
vFlag should only be provided once, to the highest level method. If no vFlag is given, false is assumed.